tests/test_indexes.py

```diff
@@ -79,12 +78,34 @@ class TestIndexes(tb.DDLTestCase):
             }]
         )
 
+        await self.con.execute(
+            """
+                ALTER TYPE test::Person
+                DROP INDEX ON ((.first_name, .last_name));
```

Comment:
Do we have a test that tests how we fail if a user tries to drop an index with an expression we couldn't find?

Reply:
Added a test.